src/connector.mjs

```diff
@@ -1,22 +1,28 @@
-import NodeWalletConnect from "@walletconnect/node";
+import SignClient from '@walletconnect/sign-client'
 import qrcode from "qrcode-terminal";
 
 const networks = {
-  1: "mainnet",
-  5: "goerli",
-  42: "kovan",
+  'mainnet': 1,
+  'goerli': 5
```

Comment:
Should we add more networks here beyond just `mainnet` and `goerli` ? 
Maybe arbitrum/arbitrum-goerli?